build.gradle.kts

```diff
@@ -51,11 +51,23 @@ allprojects {
     spotless {
         isEnforceCheck = false // Disables lint on gradle builds.
         kotlin {
-            ktlint().editorConfigOverride(mapOf("ktlint_experimental" to "enabled"))
+            ktlint().editorConfigOverride(
+                mapOf(
+                    "ktlint_experimental" to "enabled",
+                    // Disabled until https://github.com/pinterest/ktlint/pull/2273 is released
+                    "ktlint_standard_function-expression-body" to "disabled",
```

Comment:
This is a new experimental rule. In some cases it does the right thing but without the fix above it leads to compile errors. It also is rewriting a large number of files - I might just leave this disabled and manually update the simpler methods to follow the recommended style.